### DISCUSSION TITLE
Show month of selected range instead of current month

### DISCUSSION THREAD

**Discussion by gleysen** [easepick/easepick#205] [Q&A] [answered]
I have a calendar setup in 1 grid, 1 month... when I set the default selected range to a daterange in the coming month, the calendar shows this month, instead of next month. (so the date selection isn't visible)
Is there a way to make sure that the month that is shown is the month that contains the startdate of the default selected daterange?
O
So for example, we are now June... if set the date range from 7th of July until 13the of July, I would like the calendar to show the month of July, instead of now showing the month of June... and so the selected daterange isn't visible on the initial render.

**Comment by gleysen** [comment, marked as answer]
Nevermind.. found the answer! (the date option in the core setup)

**Reply by mojipo** [reply]
you looking for answer then when you found just go away and don't explain for others?! :)

**Reply by gleysen** [reply]
How so?... I did explain it in my answer above.. the date option when you create the calendar:

```
new easepick.create({
									element: document.getElementById('datepicker_new'),
									zIndex: 20,
									lang: "nl-BE",
									date: 'XXX-STARTDATE-XXX',
									grid: 2, ... and so on...
```

**Reply by mojipo** [reply]
didn't work on my case so I was wondering how you solved, but then I saw gotoDate() in the docs and problem solved ...

**Comment by eriku** [comment]
This did not fix it for me. I don't want to pre-select the minDate, I just want to have the first month shown be the month that the minDate is in whether that is the current month or a month in the future.
